addons/core/translations/actions/en-us.yaml

```diff
@@ -52,3 +52,8 @@ toggle-color-theme: Toggle Color Theme
 toggle-feature-edition: Toggle Feature Edition
 toggle-licensed-features: Toggle Licensed Features
 more: ', +{remainingItems} more'
+search: Search
+filtering:
```

Comment:
not a blocker- since this file doesn't have a nested structure, I'm wondering if we need to introduce a new pattern here-  Can we move `yes, no, narrow-results` to titles and keep search and filter here? would that work?

Reply:
Yeah that works. Keep it simple. I will update and push up.